Intersect (Core)/Enums/Directions.cs

```diff
@@ -1,9 +1,7 @@
 namespace Intersect.Enums
 {
-
     public enum Directions
```

Comment:
While you are in here can you change the name of the enum to Direction, plural is for flags (no you didn't make this mistake, I still make the mistake myself, but if you're changing the file we might as well fix it now).

Reply:
Alright 👍🏽